packages/localize/src/tools/src/extract/translation_files/format_options.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+export type FormatOptions = Record<string, string>;
+export type ValidOption = [string, string[]];
```

Comment:
We can probably use names tuples here:
```suggestion
export type ValidOption = [key: string, value: string[]];
```

Reply:
Ooh! I didn't even know that named tuples were a thing!